fix issue#98 : adding column_table_limit feature (#106)
Koihik/LuaFormatter: docs/Style-Config.md

    k3 = v3, k4 = v4
}
```
### column_table_limit

type: int, default: column_limit

> The column limit of one line for table. Default value is same as column_limit value.

Reviewer: ```suggestion
The column limit of each line of a table. Default value the same as column_limit.
```

Author: done

> ```lua

Reviewer: Change the examples to something like:
```
column_table_limit: column_limit
[...]
column_table_limit: 20
[...]
```

Author: done
